build.cmd

```diff
@@ -19,16 +19,18 @@ if not exist %NUGET_CACHE_DIR%\NuGet.exe (
 )
 
 rem copy nuget locally
-if not exist .nuget\NuGet.exe (
-  if not exist .nuget md .nuget
-  copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
-)
+if not exist .nuget md .nuget
+copy %NUGET_CACHE_DIR%\NuGet.exe .nuget\NuGet.exe > nul
 
 rem restore packages for build script
 .nuget\NuGet.exe restore .\packages.config -PackagesDirectory .\packages -Verbosity quiet
 
+rem find Visual Studio and MSBuild installation paths
+FOR /F "delims=" %%i IN ('.\packages\vswhere.1.0.62\tools\vswhere.exe -property installationPath') DO set VS_PATH=%%i

```

Comment:
If we have more than one entry, we'll keep overwriting `VS_PATH`. How do we know we're getting the right one?

Perhaps we should narrow down the version and ask for Roslyn instead of MBuild? This is closer to what we do in the build.csx.

```
vswhere.exe -nologo -latest -property installationPath -requires Microsoft.VisualStudio.Component.Roslyn.Compiler -version [15,16)
```

Even if you keep the loop, at least we'll make sure we get a VS15.